cmd/server.go

```diff
@@ -278,5 +280,13 @@ func HTTPModule() fx.Option {
 			viper.GetBool(authBearerUseScopesFlag),
 			stripe.NewLoader(),
 		),
+		cdi.ConnectorModule[dummypay.Config, dummypay.TaskDescriptor](
+			viper.GetBool(authBearerUseScopesFlag),
+			dummypay.NewLoader(),
+		),
+		cdi.ConnectorModule(
```

Comment:
@darkmatterpool I don't like this way of register connectors. It would be ideal if the connectors registers themselves to the core. If you have an idea for this....